www/modules/custom/borg_cleanup/borg_cleanup.module

```diff
@@ -58,4 +58,35 @@ function borg_cleanup_form_user_profile_form_alter(&$form, &$form_state) {
   unset($form['account']['pass']['#description']);
   $form['account']['mail']['#description'] = str_replace('A valid e-mail address. All e-mails from the system will be sent to this address. ', '', $form['account']['mail']['#description']);
   $form['account']['mail']['#description'] = str_replace(' or wish to receive certain news or notifications by e-mail', '', $form['account']['mail']['#description']);
+
+  $form['email-subscriptions'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Email Subscriptions'),
```

Comment:
I'm not sure the contact form email is a subscription. Can we call this a notification instead?